presto-main/src/main/java/com/facebook/presto/FullConnectorSession.java

```diff
@@ -166,7 +166,6 @@ public String toString()
                 .add("traceToken", getTraceToken().orElse(null))
                 .add("locale", getLocale())
                 .add("startTime", getStartTime())
-                .add("properties", properties)
```

Comment:
we dont need to delete this line

Reply:
✅

Reply:
I still see this line deleted . Can we "undelete" this line